pkg/webui/console/components/webhook-form/index.js

```diff
@@ -363,13 +480,46 @@ export default class WebhookForm extends Component {
             sensitive
             code
           />
+          <Form.Field
+            autoFocus
```

Comment:
```suggestion
```
Autofocus only makes sense for fields that are at the beginning of a form (or likely to be entered first)